HeterogeneousCore/CUDAUtilities/src/getCachingDeviceAllocator.h

```diff
@@ -5,6 +5,8 @@
 #include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
 #include "CachingDeviceAllocator.h"
 
+#include <cuda/api_wrappers.h>
```

Comment:
why do we  add this ?

Reply:
Technically the `#include` should have been there already since the code below calls `cuda::device::count()` (which I'd rather deal with in another PR). Likely one/some of the files including this header lost the `#include <cuda/api_wrappers.h>` that it got from elsewhere.